### PR TITLE
Fix issues on episode 4

### LINKED ISSUE #40: Add example solutions to discussion questions in episode4 Kanban lesson
For example, the discussion "How do you deal with a blocked item in the Kanban workflow" is missing a solution.

### LINKED ISSUE #44: Some comments on lesson 4 "Introduction to Kanban"
**Style**

- Teaching and exercise times should be fixed. 
- We can move the section `Combining Scrum and Kanban (Scrumban)` to the next lesson. 

**Text**
Perhaps we can add a discussion (like lesson 3) as:
```
Think about following the Kanban methodology when working on your projects in a team.
What questions do you still have?
Are there any incremental improvements that can benefit your team?
What did you learn in this lesson that you think is too much for your team?
```

### PR DESCRIPTION
**Pull request by SarahAlidoost**
closes #40 
closes #44

### CODE REVIEW

File: _episodes/04-kanban-intro.md [hunk 1]
```diff
@@ -146,23 +146,39 @@ the delivery point as fast as possible.
...
+>
+> > ## Solution
+> >
+> > The simplest but effective approach is to put a "red dot" on the item. In an
+> online board, it is possible to add a tag/label and more information like why
+> this item is blocked, for how long, and who will unblock it. This visualizes
+> the situation. So, the team can immediately see that the item is blocked and
+> have it under control.
+> {: .solution}
+{: .challenge}
```
Comment (svenvanderburg) [inline _episodes/04-kanban-intro.md]: Indeed this is nicer to do here as a challenge.

File: _episodes/04-kanban-intro.md [hunk 2]
```diff
@@ -146,23 +146,39 @@ the delivery point as fast as possible.
...
-But maybe a combination of Scrum and Kanban (known as "Scrumban") is what
-your team would benefit the most from.
-
-Scrumban manifests itself in different ways depending on your team
-characteristics. The team may use team meetings (like retrospective and
-planning) from Scrum and a continuous workflow with WIP limits from Kanban.
+iterations called sprints (see [episode 3](./03-scrum-into.md)). But maybe a
+combination of Scrum and Kanban (known as "ScrumBan") is what your team would
+benefit the most from. We will learn about it in the [next
+lesson](./05-comparing-agile-methodologies.md).
```
Comment (svenvanderburg) [inline _episodes/04-kanban-intro.md:175]: I like how you solved this! Indeed it was weird to have this twice in there.

File: _episodes/04-kanban-intro.md [hunk 3]
```diff
@@ -146,23 +146,39 @@ the delivery point as fast as possible.
...
 
 > ## Discussion
 >
-> Imagine that you are working with a Kanban board. One of the work-items in “In
-> Progress” got blocked. How do you deal with it in the Kanban workflow?
+> Think about following the Kanban methodology when working on your projects in a team.
+>
+> - What questions do you still have?
+> - Are there any incremental improvements that can benefit your team?
+> - What did you learn in the online course that you think is too much for your team?
```
Comment (svenvanderburg) [inline _episodes/04-kanban-intro.md:183]: Cool I like this!

File: _episodes/05-comparing-agile-methodologies.md [hunk 4]
```diff
@@ -89,6 +89,9 @@ In general failed Scrum implementations are said to have failed because of: "not
 
 Nevertheless, it is common to do a mixture of Scrum and Kanban, picking the best of both worlds according to the team's needs.
 This is called ScrumBan.
+ScrumBan manifests itself in different ways depending on your team
+characteristics. The team may use team meetings (like retrospective and
+planning) from Scrum and a continuous workflow with WIP limits from Kanban.
```
Comment (svenvanderburg) [inline _episodes/05-comparing-agile-methodologies.md:94]: Nice that you added this example here.

File: _episodes/04-kanban-intro.md [hunk 5]
```diff
@@ -146,23 +146,39 @@ the delivery point as fast as possible.
...
+> continue working on it because it is waiting for something from outside the
+> team. How do you deal with the blocked item in the Kanban workflow?
+>
+> > ## Solution
+> >
+> > The simplest but effective approach is to put a "red dot" on the item. In an
+> online board, it is possible to add a tag/label and more information like why
+> this item is blocked, for how long, and who will unblock it. This visualizes
+> the situation. So, the team can immediately see that the item is blocked and
+> have it under control.
```
Comment (svenvanderburg) [inline _episodes/04-kanban-intro.md:162]: Would be good to add this I think:
```suggestion
> have it under control.
> Now, the blocked item does not count as a WIP item and 
> the team could potentially add an extra work-item to 'In Progress'.
```

File: _episodes/04-kanban-intro.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Introduction to Kanban"
-teaching: 5
-exercises: 60
+teaching: 15
+exercises: 30
```
Comment (svenvanderburg) [inline _episodes/04-kanban-intro.md:4]: This timing makes more sense indeed.

**Review by svenvanderburg — APPROVED** [review]
Looks good @SarahAlidoost :) I added one small suggestion.

**Comment by SarahAlidoost** [comment]
> Looks good @SarahAlidoost :) I added one small suggestion.

Thanks for reviewing :)

**Pull request merged**